package.json

```diff
@@ -25,9 +25,15 @@
   },
   "dependencies": {
     "@opensecret/react": "0.3.0",
+    "@radix-ui/react-accordion": "^1.2.1",
```

Comment:
pls lock all deps to exact versions. btw to do that when adding packages just use --exact flag so `bun add motion --exact` for example

Reply:
motion is added with --exact. Can you do the same for using the shadcn cli? I noticed you weren't pinning exact versions for that? Otherwise I can just manually do it

Reply:
not sure. if I didn't do it, I forgot